internal/business/persisted_operations/gcp_storage_loader.go

```diff
@@ -80,5 +88,13 @@ func (g *GcpStorageLoader) Load(ctx context.Context) error {
 		_ = reader.Close()
 	}
 
+	log.Info(fmt.Sprintf("Read %d manifest files from bucket", numberOfFilesProcessed))
+
 	return errors.Join(errs...)
 }
+
+func getFileName(attrs *storage.ObjectAttrs) string {
+	var fileNameSplit = strings.Split(attrs.Name, "/")
```

Comment:
could you define variables with an instantion as `varname := value`
If you want a default-value variable you can do `var varname type`